devops/girder.Dockerfile

```diff
@@ -0,0 +1,52 @@
+
+FROM node:latest as web-builder
+
+WORKDIR /app
+
+# Install dependencies
+COPY client/package.json client/yarn.lock /app/
+RUN yarn --frozen-lockfile
+
+# Build client
+COPY client/ /app/
+RUN yarn build
+
+FROM girder/girder:latest-py3 as runtime
+
+# Delcare Environment Variables
+ENV GIRDER_MONGO_URI mongodb://mongo:27017/girder
+ENV GIRDER_ADMIN_USER admin
+ENV GIRDER_ADMIN_PASSWORD upenn
+ENV CELERY_BROKER_URL amqp://guest:guest@rabbit/
+ENV BROKER_CONNECTION_TIMEOUT 2
+
+# TODO: explain why this is necessary
+# Install Large Image from source instead of pip because ?
```

Comment:
I'd like to have a better comment here.

Reply:
This was necessary early on in the project because nd2 support was still in a branch or large_image.  We should be able to use it from pypi (but probably want the --pre flag, since I think we have benefit from some changes that haven't been officially released let).